### ISSUE TITLE
Settings

### ISSUE THREAD

**Issue by noyainrain** [noyainrain/meetling#3]
Add (site) settings. For now, admins should be able to adjust the branding:
- title
- icon URL
- favicon URL

With this feature, implement first part of auto update functionality. This depends on #5 .

**Comment by noyainrain** [comment]
Draft:

``` python
class Settings(Object):
    """
    Global settings.

    .. describe:: id

       Unique ID ``settings``.

    .. describe:: title

       Site title.

    .. describe:: icon

       Site icon.

    .. describe:: favicon

       Site icon optimized for a small size.

    .. http:get:: /api/settings

       Get the settings.
    """

class Meetling:
    """
    .. attribute:: settings

       Global settings.
    """
```

**Comment by noyainrain** [comment]
Mockup: https://app.moqups.com/NoyaInRain/zqvfwQiAkh/view/page/aa4f3edbb

**Comment by noyainrain** [comment]
Auto update draft:

``` python
class Meetling:
    def update(self):
        """Update the Meetling database.

        If the database is fresh, it is initialized. If the database is already up-to-date, nothing
        is done. Thus it is safe to call the method without knowing if an update is necessary.
        """
```

**Comment by noyainrain** [comment]
Implemented in 4aed44def7b3bde433d0071b2e18c4cb089c41d8 .
